duet/mr1DEVELOPMENT/views/dev_desktop_install_partners.view.lkml

```diff
@@ -0,0 +1,194 @@
+view: dev_desktop_install_partners {
+  derived_table: {
+    sql:
+    WITH tbl AS
```

Comment:
We should probably move this into bqetl and just reference the table here.

Reply:
I'm just restoring these files for Su. These were how the files were before we deleted them due to deprecation. No, it's not done in the style that we would want as they were created a long time ago before we had the standard method.